test/core/bitswap.spec.js

```diff
@@ -67,10 +65,11 @@ function connectNodes (remoteNode, inProcNode, callback) {
 
 let nodes = []
 
-function addNode (inProcNode, callback) {
+function addNode (fDaemon, inProcNode, callback) {
   fDaemon.spawn({
     exec: './src/cli/bin.js',
     initOptions: { bits: 512 },
+    port: 0,
```

Comment:
No need for this line. Already caught a bunch of these. Mind checking everywhere?

Reply:
Yeah, doing the check right now. Thanks!